eng/Publishing.props

```diff
@@ -2,6 +2,60 @@
 
   <PropertyGroup>
     <ProducesDotNetReleaseShippingAssets>true</ProducesDotNetReleaseShippingAssets>
+    <PublishDependsOnTargets>$(PublishDependsOnTargets);_PublishInstallers</PublishDependsOnTargets>
   </PropertyGroup>
 
+  <ItemGroup>
```

Comment:
Please hoist the condition into the <ItemGroup>

Reply:
I presume the ask is to move the condition from individual items to the group - `Condition="'$(DotNetBuildRepo)' == 'true'"`. I'll do that.

Reply:
Fixed with https://github.com/dotnet/runtime/pull/99432/commits/18768165afb5bd3020cf0e6cc56c104265cdc7ba